lib/debugmenu.js

```diff
@@ -41,182 +49,50 @@ class DebugMenu extends LitElement {
 
   static get properties () {
     return {
-      debugEntries: { type: Object },
       isOpen: { type: Boolean },
-      cursorBlock: { type: Object }
+      cursorBlock: { type: Object },
```

Comment:
The entire update sequence holds on changing cursorBlock, nothing updates if you look into the sky

Reply:
indeed, fixed